.cbmc-batch/jobs/Makefile.local_default

```diff
@@ -28,4 +28,7 @@ COMMON_HELPERDIR ?= $(COMMONDIR)/.cbmc-batch
 SDKDIR ?= $(abspath ../../..)
 HELPERDIR ?= $(SDKDIR)/.cbmc-batch
 
-addone = $(shell echo $$(( $(1) + 1)))
\ No newline at end of file
+CWD = $(abspath .)
+GOTO_DIR = $(CWD)/gotos
```

Comment:
Looks like you're using GOTO_DIR here but GOTODIR in the Makefiles within the individual proof directories (e.g. .cbmc-batch/jobs/aws_cryptosdk_aes_gcm_decrypt/Makefile). Should these match?